sandbox/apparmor/apparmor.go

```diff
@@ -366,6 +366,12 @@ func probeKernelFeatures() ([]string, error) {
 			}
 		}
 	}
+	if data, err := os.ReadFile(filepath.Join(rootPath, featuresSysPath, "policy", "permstable32")); err == nil {
+		permstableFeatures := strings.Fields(string(data))
+		for _, feat := range permstableFeatures {
+			features = append(features, fmt.Sprintf("policy:permstable32:%s", feat))
+		}
+	}
 	return features, nil
```

Comment:
this needs a sort.Strings(features) before returning

Reply:
Ahhhh, I didn't notice the importance of `// note that os.ReadDir() is already sorted`